planet4-gutenberg-blocks.php

```diff
@@ -104,10 +104,12 @@
 // pages allow all block types.
 const PAGE_BLOCK_TYPES = [
 	'planet4-blocks/articles',
+	'planet4-blocks/carousel-header',
 	'planet4-blocks/columns',
 	'planet4-blocks/cookies',
 	'planet4-blocks/counter',
 	'planet4-blocks/covers',
+	'planet4-blocks/enform',
```

Comment:
@pablocubico can you remove enform from here. better allow it in the repo that the block is implemented.

@sagarsdeshmukh has opened a PR for this
https://github.com/greenpeace/planet4-plugin-gutenberg-engagingnetworks/pull/6

Reply:
Oh right, I saw his comment on the chat. 👍

Reply:
Updated.